tsconfig.json

```
@@ -23,19 +19,15 @@
     ],
     "baseUrl": ".",
     "paths": {
-      "@/*": [
-        "./src/*"
-      ]
+      "@/*": ["./src/*"],
+      "@/components/containers/*": ["./src/components/containers/*"],
+      "@/components/containers/homepage/*": [
+        "./src/components/containers/homepage/*"
+      ],
+      "@/components/ui/*": ["./src/components/ui/*"]
```

Tous ceux la sont inutiles, laisse juste celui de @/*, les autres ne servent a rien à moins que tu fasse par example : 

```suggestion
      "@containers/*": ["./src/components/containers/*"],
      "@homepage/*": [
        "./src/components/containers/homepage/*"
      ],
      "@ui/*": ["./src/components/ui/*"]
```

Mais mieux vaut pas franchement, supprime les juste